apps/app/src/configs/fulfillment.ts

```diff
@@ -0,0 +1,42 @@
+export const fulfillmentConfig = {
+  default: {
+    sendOrder: true,
+    pickUp: true,
+    mailOrder: false,
+    mailOrderProviders: [
+      ...(process.env.REACT_APP_CUREXA_ORG_ID ? [process.env.REACT_APP_CUREXA_ORG_ID] : [])
+    ],
+    sendToPatient: false,
+    sendToPatientUsers: [] // its weird to have this here, but its temporary and keeps ts from complaining
+  },
+  // Weekend Health
+  [process.env.REACT_APP_WEEKEND_ORG_ID as string]: {
+    sendOrder: false,
+    pickUp: true,
```

Comment:
One thing we lost was `disableOrderCreation`, could we disable pickUp here as well for weekend health? As they only want to create orders programatically, so maybe the component can just show an error if there are no valid options

Reply:
Wait nvm, just noticed `sendOrder`, ignore me